cli/cmd/sync.go

```diff
@@ -233,6 +234,11 @@ func sync(cmd *cobra.Command, args []string) error {
 		if err != nil {
 			return enrichClientError(managedplugin.Clients{}, []bool{source.RegistryInferred()}, err)
 		}
+		latestVersion, err := sourcePluginClient.FindLatestPluginVersion(ctx, managedplugin.PluginSource)
+		if err == nil && source.Version != "" && latestVersion != "" && semver.IsLess(source.Version, latestVersion) {
+			log.Warn().Str("source", source.Name).Msgf("Your source plugin is out of date, you are using version %s, and the latest version is %s. Consider upgrading.", source.Version, latestVersion)
```

Comment:
Could (or should) we also include a link to the hub? Like:

```suggestion
			log.Warn().Str("source", source.Name).Str("url", "https://hub.cloudquery.io/plugins/source/" + source.Path).Msgf("Your source plugin is out of date, you are using version %s, and the latest version is %s. Consider upgrading.", source.Version, latestVersion)
```

Reply:
Fixed.